finagle-core/src/test/scala/com/twitter/finagle/util/LoadServiceTest.scala

```diff
@@ -94,6 +95,12 @@ class LoadServiceTest extends FunSuite with MockitoSugar {
     executor.shutdown()
   }
 
+  test("LoadService should ignore packages according to ignoredPaths GlobalFlag") {
```

Comment:
I think I see why it doesn't work.
We are caching the classes meta-data per classloader, and LoadService has already scanned the classpath in a previous test.
Maybe adding a method to flush the cache (private to the package) or setting a new Classloader would solve the problem.